File: LLM-Translations/GPT-4/codenet/Go/C/s033683118.c

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int getNextInt() {
    int i;
    scanf("%d", &i);
    return i;
}

int main() {
    int k, x;
    k = getNextInt();
    x = getNextInt();

    char ans[4] = "No";
    if (k * 500 >= x) {
        strcpy(ans, "Yes");
    }
    printf("%s\n", ans);

    return 0;
}

File: LLM-Translations/GPT-4/codenet/Go/Python/s033683118.py
import sys
import os
from typing import Any, Tuple, List

def get_next_string(scanner: List[str]) -> str:
    return scanner.pop(0)

def get_next_int(scanner: List[str]) -> int:
    return int(get_next_string(scanner))

def get_next_int64(scanner: List[str]) -> int:
    return int(get_next_string(scanner))

def get_next_uint64(scanner: List[str]) -> int:
    return int(get_next_string(scanner))

def get_next_float64(scanner: List[str]) -> float:
    return float(get_next_string(scanner))

def solve(scanner: List[str]) -> str:
    k = get_next_int(scanner)
    x = get_next_int(scanner)

    ans = "No"
    if k * 500 >= x:
        ans = "Yes"
    return ans

def main() -> None:
    scanner = sys.stdin.read().split()
    print(solve(scanner))

if __name__ == "__main__":
    main()


